### DISCUSSION TITLE
When will 2.8.4 be released

### DISCUSSION THREAD

**Discussion by chensunny** [Kong/kong#11085] [Help] [answered]
2.8.x branch fixes issue #9090  to resolve the process restart configuration loss. When can the request be released.

**Comment by git-hulk** [comment]
Hi, we also suffer from this issue a lot. Would you please release the 2.8.x for us?

**Reply by gruummy** [reply]
Same problem in our company. We also can not so fast switch to kong version 3

**Reply by git-hulk** [reply]
https://github.com/Kong/kong/releases/tag/2.8.4 @Gruummy 2.8.4 has been released.

**Reply by telekom-digioss** [reply]
Interesting...
all the interisting changes / fixes which already had been exisiting in the 2.8.4 preparation branch  had been reverted.
So this "new" 2.8.4 has much less contend compared with the in the past released patch version of the enterprise version

**Reply by git-hulk** [reply]
Yes, I see the ChangeLog contains only one bug fix. 😢

**Reply by guanlan** [reply, marked as answer]
In general, we don't backport bugfix to OSS versions. We would recommend to upgrade the lastest 3.x.

**Reply by git-hulk** [reply]
@guanlan Thanks for your reply.
